Cannot run make build

## Describe the bug
Getting `invalid version` error on `github.com/ChainSafe/go-schnorrkel@v0.0.0-20210127175223-0f934d64ecac` during `make gossamer`


## Expected Behavior

Should build

## Current Behavior

Throws the following error

`go: github.com/ChainSafe/go-schnorrkel@v0.0.0-20210127175223-0f934d64ecac: invalid version: unknown revision 0f934d64ecac`

## Possible Solution

Use the correct version

## To Reproduce
Steps to reproduce the behavior:

1. Clone the repo 
2. run `make gossamer`


## Log output

$ make gossamer
```
rm -fr ./bin
cd cmd/gossamer && go build -o ../../bin/gossamer && cd ../..
go: github.com/ChainSafe/go-schnorrkel@v0.0.0-20210127175223-0f934d64ecac: invalid version: unknown revision 0f934d64ecac
```

## Specification

- go version: 1.15.2
- gossamer version: development

I got the same error here after running `go clean --modcache` 

```
> go clean --modcache
> make gossamer
rm -fr ./bin
cd cmd/gossamer && go build -o ../../bin/gossamer && cd ../..
go: github.com/ChainSafe/go-schnorrkel@v0.0.0-20210127175223-0f934d64ecac: invalid version: unknown revision 0f934d64ecac
make: *** [Makefile:125: gossamer] Error 1
```

--- a/go.mod
+++ b/go.mod
@@ -2,7 +2,7 @@ module github.com/ChainSafe/gossamer
 
 require (
 	github.com/ChainSafe/chaindb v0.1.5-0.20210117220933-15e75f27268f
-	github.com/ChainSafe/go-schnorrkel v0.0.0-20210127175223-0f934d64ecac
+	github.com/ChainSafe/go-schnorrkel v0.0.0-20210222182958-bd440c890782
 	github.com/ChainSafe/log15 v1.0.0
 	github.com/OneOfOne/xxhash v1.2.5
 	github.com/btcsuite/btcutil v1.0.2

